src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -38,7 +39,7 @@
  *
  * @property array<string, array{string, bool, string, null}> $insertionMap
  */
-final class BetterStandardPrinter extends Standard
+final class BetterStandardPrinter extends Printer
```

Comment:
Why this change? We should relly on basic printer, to avoid unnecesary coupling.

Reply:
That's per @ondrejmirtes suggestion above https://github.com/rectorphp/rector-src/pull/4305#issuecomment-1598825994

Reply:
without extends PHPStan printer, we got error:

```
undefined method pPHPStan_Node_AlwaysRememberedExpr
```

per PR description.